src/orders/dto/order.create.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import { IsArray, IsMongoId, IsString, ValidateNested } from 'class-validator';
+import { Types } from 'mongoose';
+
+export class orders {
+  @ApiProperty({ type: Types.ObjectId })
+  @IsMongoId()
+  menu: Types.ObjectId;
+  @ApiProperty({ type: [Types.ObjectId] })
+  @IsArray()
+  @IsMongoId({ each: true })
+  addons: Types.ObjectId[];
```

Comment:
I think the type is `string[]` because when you response to frontend id is string (from requirement)